storybookjs/storybook
# UI: Add initialActive option parameter

Issue: [#9127](https://github.com/storybookjs/storybook/issues/9127)

## What I did

- Added `initialActive` option parameter to storybook-ui config.
- Active tabs in Mobile layout are now handled using constants instead of hardcoded numbers.
- Updated `Options Parameters` doc page.

Users can now set the default active tab on Mobile.

## How to test

To test this feature, simply set `initialActive` in the options of the storybook config, and check that the default active tab on Mobile matches your setup:

```javascript
// config.js
...

/**
 * 'sidebar' - Sidebar
 * 'canvas' - Canvas
 * 'addons' - Addons
 */
const initialActive = X // one of above values
addParameters({
  options: {
     initialActive,
  },
})
```
